fuel/app/classes/model/user.php

```diff
@@ -94,7 +94,7 @@ static public function find_by_name_search($name)
 		$user_table = \Model_User::table();
 		$matches = \DB::select()
 			->from($user_table)
-			// Do not return super users or the current user
+			// Do not return super users or the current user // << why?
```

Comment:
This is because:
- superusers aren't supposed to be visible to normal users (the superuser account isn't associated with a UCF identity)
- Previously, the only time this search was run was in the context of sharing a widget with someone, so there was no need to search for yourself. With the addition of instance admin, it makes sense to include the current user in the search results (so you can add yourself to another widget, for example), so this could probably be changed. That's not really within the scope of this PR though.